### ISSUE TITLE
Help: ...ERROR not found

### ISSUE THREAD

**Issue by hcp1227** [qdm12/ddns-updater#292]
**TLDR**: *Describe your issue in a one liner here*

1. Is this urgent: No
2. DNS provider(s) you use: dnspod
3. Program version:

    

   running version latest built on 2021-12-15T09:57:37Z (commit ae7f104)

4. What are you using to run the container: unraid
5. Extra information (optional)

Logs:
INFO IPv6 address of unraid.xwhs123.com is 2409:8a55:8035:7310:4cdd:66ff:fe7a:c1f0 and your IPv6 address is 2409:8a55:803a:a740:4cdd:66ff:fe7a:c1f9
2022/01/21 08:46:42 INFO Updating record [domain: xs123.com | host: unraid | provider: dnspod | ip: ipv6] to use 2409:8a55:803a:a740:4cdd:66ff:fe7a:c1f9
2022/01/21 08:46:43 ERROR not found

Host OS:unraid

**Comment by qdm12** [comment]
Thanks for reporting, yes the IPv6 behavior of the code was wrong.

Fixed in f5b76f214e486eae27ba3cdd9d92429193639900

**Comment by hcp1227** [comment]
there is still a problem
 
http server: listening on :8000
2022/01/21 23:19:35 INFO healthcheck server: listening on 127.0.0.1:9999
2022/01/21 23:19:35 INFO IPv6 address of unraid.xwhs123.com is 2409:8a55:8035:7310:4cdd:66ff:fe7a:c1f0 and your IPv6 address is 2409:8a55:803a:a740:4cdd:66ff:fe7a:c1f9
2022/01/21 23:19:35 INFO Updating record [domain: xwhs123.com | host: unraid | provider: dnspod | ip: ipv6] to use 2409:8a55:803a:a740:4cdd:66ff:fe7a:c1f9
2022/01/21 23:19:36 ERROR malformed IP address received:

**Comment by qdm12** [comment]
@hcp1227 do you still have the issue on the latest version? I re-checked their API documentation and it seems to match today's code. Please ideally create another issue if you have a problem still. Thanks!
